pkg/chain/ethereum/ethereum.go

```diff
@@ -130,6 +130,18 @@ func newBaseChain(
 		)
 	}
 
+	if config.Network != ethereum.Developer &&
```

Comment:
Don't we need to check against `Unknown` network as well?

Reply:
We don't expect to receive and handle the `Unknown` network anywhere. The way the `resolveEthereumNetwork` works it will always fallback to `Mainnet`.
https://github.com/keep-network/keep-core/blob/6a8e601861ad935a6cc5b1ae381d62919b38a042/config/config.go#L55-L70

Reply:
This functions has changed and we do handle the `Unknown` network https://github.com/keep-network/keep-core/blob/main/config/config.go#L56-L80

Reply:
Nice catch! Yeah, but if we receive the `Unknown` network here clearly something is wrong with the configuration, so the client should fail to start anyway. I don't think we should handle the `Unknown` network case, if the client haven't failed up to this point it should fail now.